CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing Guidelines
+
+Thank you for your interest in contributing to the vscode-dbt-power-user extension! We greatly appreciate the dedication of our community members in helping us enhance this project.
+
+## Bug Reports and Feature Requests
+
+One of the simplest ways to contribute is by sharing your thoughts with us. If you come across any bugs or have ideas for new features or improvements, please start a discussion in our dedicated channel: `#tools-dbt-power-user` on the [dbt community Slack](https://www.getdbt.com/community/join-the-community/). Additionally, take a look at the [GitHub issues](https://github.com/innoverio/vscode-dbt-power-user/issues) to see if the topic has already been raised. If not, feel free to create a new issue. Your detailed bug reports, along with steps to reproduce the issue and relevant environment information, are invaluable to us and other contributors.
```

Comment:
You can make a Slack link that links directly to the channel.

Reply:
added link to the direct channel.